CONTRIBUTING.md

```diff
@@ -1,20 +1,28 @@
 # Contributing
 
 - Creating a bundle: `npm prepublish`
-- Working on docs: `npm run docs:watch`
-- Developing with tests: `npm run test:watch`
+- Working on docs: `npm run docs:watch` / `yarn docs:watch`
+- Developing with tests: `npm run test:watch` / `yarn test:watch`
+- Fixing js and ts code style: `npm run style:fix` / `yarn style:fix`
+- Commiting (with commitizen): `npm run commit` / `yarn commit`
 
 ## Committing
 
-Ensure you have [commitizen](https://github.com/commitizen/cz-cli) installed and commit using `git cz`.
+We are using semantic-release and conventional-changelog for releasing, so our commit messages have to follow strict rules.
+
+The commit message formatting can be added using a typical git workflow or through the use of a CLI wizard ([Commitizen](https://github.com/commitizen/cz-cli)).
+
+To use the wizard, run `npm run commit` or `yarn commit` in your terminal after staging your changes in git.
```

Comment:
hard to say, but I don't think so, if the commit message is cloberet it will not throw, instead will generate not the changelog that we were looking for.

Anyway thanks for pointing this out, I did some googling and there is this new kid on the block, which may be what we want ( incorporated in `npm test` )

https://github.com/marionebl/conventional-changelog-lint

```
{
    "scripts": {
      "test": "conventional-changelog-lint --from=HEAD~1"
    }
  }
````